legion/engine/core/defaults/hierarchysystem.hpp

```diff
@@ -11,6 +11,10 @@ namespace legion::core
         void onRotationModified(events::component_modification<rotation>* event);
         void onScaleModified(events::component_modification<scale>* event);
 
+        void onPositionBulkModified(events::bulk_component_modification<position>* event);
+        void onRotationBulkModified(events::bulk_component_modification<rotation>* event);
+        void onScaleBulkModified(events::bulk_component_modification<scale>* event);
```

Comment:
why are all these modify functions in past tense?

Reply:
because they get called after they have been modified